src/components/Comments/Comment.js

```diff
@@ -46,6 +55,15 @@ const Comment = ({
     level,
     updateVote
   });
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of a loading state and context usage are good practices for managing state within a component. However, error handling should be considered when using hooks like `useCreateChat` and `useDMMessage`. If these hooks fail, the component should be able to handle those errors gracefully.

```diff
+ const [loading, setLoading] = React.useState({
+   loadingDm: false,
+   loadingDmAnon: false
+ });
+ 
+ const [profile] = React.useContext(Context).profile;
+ const {createSignChat} = useCreateChat();
+ const {sendMessageDM} = useDMMessage();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->